### PR TITLE
Add a new built-in function `get_working_dir()`

### LINKED ISSUE #452: pwd() interpolation helper
hi,

Following the additions of hooks in terragrunt, I'm looking for a way to use the absolute location where the hooks are executed to manage substitutions of vars inside a *.tfvars file located right inside terragrunt's tmp dir.

Something like a `pwd()` helper.
Also, it might not be as easy to implement as given my understanding of terragrunt sequences, interpolations are resolved before copying files in the tmp dir?

How would you solve this?

### PR DESCRIPTION
**Pull request by levkohimins**
## Description

Add a new built-in function `get_working_dir()`

Fixes #452

### CODE REVIEW

File: docs/_docs/.DS_Store
Comment (denis256) [inline docs/_docs/.DS_Store:1]: this file is required?

Comment (levkohimins) [reply docs/_docs/.DS_Store:1]: Ah, no. Removed.

**Review by denis256 — APPROVED** [review]

**Comment by levkohimins** [comment]
@denis256 , thanks for the review!

**Review by denis256 — APPROVED** [review]

**Comment by levkohimins** [comment]
@arsci Thank you!

**Comment by levkohimins** [comment]
Resolved in [v0.53.4](https://github.com/gruntwork-io/terragrunt/releases/tag/v0.53.4) release.

**Review by binabdulkhalim-prog — COMMENTED** [review]
c4dcbf1d385ce525fa9113298cb7f370885d99cd

**Pull request merged**